### PR TITLE
Ignore 'nightly' tag in the version detection script

### PR DESCRIPTION
**Pull request by victoryforce**

### CODE REVIEW

File: tools/get_git_version_string.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-VERSION="$(git describe --tags --dirty)"
+VERSION="$(git describe --tags --dirty --exclude nightly)"
```
[the remark this answers is not in the document]
Comment (TurboGit) [reply tools/get_git_version_string.sh:3]: Indeed --match option is better for the future.

[the remark this answers is not in the document]
Comment (victoryforce) [reply tools/get_git_version_string.sh]: Yes, that's a rational thought. Fixed.

**Review by TurboGit — APPROVED** [review]
Thanks!

**Comment by kmilos** [comment]
Turns out this is also needed when Windows & macOS artifacts are created in the nigtly workflow: https://github.com/darktable-org/darktable/pull/14677

**Comment by victoryforce** [comment]
@kmilos My apologies, I didn't notice that there are places that use their own code with homemade regex instead of calling get_git_version_string.sh. Actually, the correct and future-proof fix would be to deduplicate code and replace them with a call to get_git_version_string.sh.

**Pull request merged**